README.md

```diff
@@ -5,14 +5,16 @@
 
 * **Documentation**: [https://aws.github.io/copilot-cli/](https://aws.github.io/copilot-cli/)
 
-The AWS Copilot CLI is a tool for developers to build, release and operate production ready containerized applications on Amazon ECS and AWS Fargate.
+The AWS Copilot CLI is a tool for developers to build, release and operate production ready containerized applications 
+on AWS App Runner, Amazon ECS, and AWS Fargate.  
```

Comment:
```suggestion
The AWS Copilot CLI is a tool for developers to build, release and operate production ready containerized applications on AWS App Runner, Amazon ECS, and AWS Fargate.  
```

Reply:
Or is this intentional?

Reply:
haha yea it was just so that this line would fit in my screen :D